dev/domain-controller/README.md

```diff
@@ -74,9 +68,40 @@ The following steps outline the procedure for performing a deployment of CAM usi
 12. Read the Terms and Conditions and once you are satisified with the information you have entered click the <b> I Agree</b> icon.
 13. Click <b>Purchase</b> to begin deployment.
 
-The deployment will now begin to run. You can track it through the notifications icon or for a more detailed view of your deployment click the <b>Resource Groups </b> icon located on the left hand side of the page and click on your resource group.
+The deployment will now begin to run. 
+<b>NOTE:</b> In general it takes over an hour to complete deployment.
```

Comment:
Not for this pull request but I think if we're going Markdown we should try to be more pure Markdown. with **text** being bold I think and no <p> or <h3's>, etc.